test/graphql/types/Organization/updater.test.ts

```
@@ -0,0 +1,260 @@
+			jwt: {},
+			pubsub: {},
+		} as TestContext;
+	});
+
+	it("should throw forbidden error when current user is not found", async () => {
+		ctx.drizzleClient.query.usersTable.findFirst.mockResolvedValueOnce(
+			undefined,
+		);
+
+		await expect(
+			OrganizationResolver.updater(mockOrganization as any, {}, ctx as any),
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)